readthedocs/projects/tasks/__init__.py

```diff
@@ -0,0 +1,27 @@
+# TODO: remove this file completely
+#
+# We only need this file to accept tasks triggered by old web instances during
+# deploy. Once it's deployed, we can delete it completely since new web
+# instances will trigger the new task
+# `readthedocs.projects.tasks.builds.update_docs_task` instead
+#
+# We may want to deploy builders first, since we will be triggering the new
+# task from webs and old builders won't have it defined otherwise.
```

Comment:
Actually, I realized this when I spun up an instance with this code. `webs` are currently triggering `readthedocs.projects.tasks.update_docs_task` which was not registered in my branch. Adding this code, solved that problem.

However, when we finally deploy this, if we deploy `webs` first they will start triggering task named as `readthedocs.projects.tasks.builds.update_docs_task` and old instances of builders won't have it registered. So, we should probably deploy builders first 🤔  --or overwrite the names of the task to keep the old name _and_ the new name at the same time.

Reply:
Actually, I think there may be other tasks that will be affected by this. Those that are executed by `web-celery` as well. I'm thinking on `fileify` too.

Reply:
https://docs.celeryproject.org/en/stable/userguide/tasks.html#names

Reply:
I updated the code in this commit a393ddbb664c25b9e211517d5049e5f2297f4959. Let me know if I was clear enough and if it does make sense to you.

Reply:
The code mostly makes sense, but I might be a little lost now on what we would need to do to test this. It seems this commit solves the issue of task renaming entirely for now, correct? That is, to start an instances with this branch on a new instance, there is no special set up required?

Reply:
No, I solved it. The instance is running code now and attending tasks triggered by "old webs instances" using the "old task name". When they receive it, they re-trigger the task using the new name and passing the proper parameters (in case the signature has changed)

However, note that the commit that I linked was not correct and the newer solution was a little different. See these commits: 9154e8d363bc6843697a38268f8d51c81bb44bb4 and 1e9c7240251ba0e67c2945da07172804b2ee32cc